content/blog/2020-09-09-september-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,191 @@
+---
+title: September '20 Heartbeat
+date: 2020-09-09
+description: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+descriptionLong: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+picture: 2020-09-09/header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/september-20-heartbeat/488
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - R
+  - Meetup
+  - Videos
+---
+
+Welcome to our September roundup of cool news, new releases, and recommended
```

Comment:
It feels we can skip this paragraph? It feels repetitive ... may be even next two titles? (not sure)

Reply:
yeah, let's skip it. it felt silly writing it, i think that's a sign it's repetitive.